bun/whatsapp-with-vonage/src/utils.ts

```diff
@@ -0,0 +1,21 @@
+//  Returns the contents of a file in the static folder
+
```

Comment:
Please change this comment to match the other TS templates https://github.com/appwrite/templates/blob/6dab35f1f221ba075640d93f5a8e374f4fba4349/deno/whatsapp-with-vonage/src/utils.ts#L2